pybv/io.py

```diff
@@ -491,5 +492,11 @@ def _write_bveeg_file(eeg_fname, data, orientation, format, resolution, units):
         raise ValueError(msg)
     data = data.astype(dtype=dtype)
 
+    # Swap bytes if system architecure is big-endian and dtype is "native" or
+    # "big-endian"
+    if ((sys.byteorder == "big" and data.dtype.byteorder == "=") or
+            data.dtype.byteorder == ">"):
+        data = data.byteswap()
```

Comment:
If you want to avoid indentation you could define `byteorder = data.dtype.byteorder`, then the `if` condition fits in one line.

```suggestion
    byteorder = data.dtype.byteorder
    if (sys.byteorder == "big" and byteorder == "=") or byteorder == ">":
        data = data.byteswap()
```